### DISCUSSION TITLE
Ikoner fra Flyt

### DISCUSSION THREAD

**Discussion by Slaine2** [fremtind/jokul#1971] [Annet]
@CamillaDahlstroem @Murstam Jeg så at det er ikoner fra flyt.

https://www.figma.com/file/TkbB9ANfejDSjB2u4u1OEuqM/J%C3%B8kul-komponenter-byggesone?node-id=12225%3A33268

Er disse i bruk? skal disse muligens inngå i biblioteket?

**Comment by Murstam** [comment]
Hvis de er i bruk så er det kanskje noen av dem flere har behov for. F eks søk og dokument?
